### DISCUSSION TITLE
Persona bar users/roles SendEmail checkbox

### DISCUSSION THREAD

**Discussion by GTSageDev** [dnnsoftware/Dnn.Platform#4319] [Ideas]
Would it be possible for the SendEmail checkbox to not be reset each time a user's role is changed (in Users), or each time a user is added to / removed from a role (in Roles). 

Recently, I had to change a couple of hundred users' roles, I didn't want emails sent, so it meant clearing the check box each time.

The code is [here](https://github.com/dnnsoftware/Dnn.Platform/blob/80e48fcde4694d70960253d401a8fcd6851608eb/Dnn.AdminExperience/ClientSide/Users.Web/src/_exportables/src/components/UserTable/UsersRoles/index.jsx#L20) (Thanks Paras Daryanani!)

Presumably it would mean moving the SendEmail property up a level - but I'm a beginner at React!

**Comment by valadas** [comment]
Well yeah, it's not impossible for this preference to be remembered in between. Would have to look at this better but either we add some state that persists between usage I guess...
